### PR TITLE
Added composer.json install install instructions and ported the docs to the wiki

### PR DESCRIPTION
**Pull request by Hackzzila**
I like it when repos also have the config for composer.json

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ To install, run the following (assuming composer is in your PATH)
 ```sh
 composer require cleanse/discord-hypertext
 ```
+Or put this in your coposer.json file
```
Comment (cryptiklemur) [inline README.md]: `composer.json`

File: README.md [hunk 2]
```diff
@@ -22,6 +22,14 @@ To install, run the following (assuming composer is in your PATH)
 ```sh
 composer require cleanse/discord-hypertext
```
Comment (cryptiklemur) [inline README.md]: should change this to `composer require cleanse/discord-hypertext dev-master` while you are editing this file

**Comment by Hackzzila** [comment]
Ok, will do :D, I also am porting over the docs to the wiki if you didn't notice.

**Comment by Hackzzila** [comment]
It is done :D

File: README.md [hunk 3]
```diff
@@ -20,7 +20,15 @@ Changes:
 To install, run the following (assuming composer is in your PATH)
 
 ```sh
-composer require cleanse/discord-hypertext
+composer require cleanse/discord-hypertext dev-master
+```
+Or put this in your coposer.json file
```
Comment (cryptiklemur) [inline README.md]: Still missed this spelling mistake

**Comment by Hackzzila** [comment]
I will fix it when I get home

**Comment by Hackzzila** [comment]
What spelling mistake?

**Comment by cryptiklemur** [comment]
`coposer.json`

**Comment by Hackzzila** [comment]
Whoops

**Comment by Hackzzila** [comment]
I fixed it and removed the examples and docs because they are now on the wiki.

**Comment by Hackzzila** [comment]
I cloned the original docs to a new repo and put the link in the wiki.
https://github.com/Hackzzila/discord-hypertext-docs

**Pull request closed without merging**